test/interval_tests/non_BigFloat.jl

```diff
@@ -43,7 +43,7 @@ end
     @test f(c) == Interval(0.19999999999999998, 0.30000000000000004)
 
     d = @interval(0.1, 0.2)
-    @test f(d) == @biginterval(0.2, 0.3)
+    @test f(d) == Interval(big"0.200000000000000005546", big"0.300000000000000005567")
```

Comment:
`@round` macro returns this value in `+` function.

Reply:
The result is wrong, since that interval does not contain the rational 2//10...

Reply:
This is strange. I have only made `+` call the function in `IntervalArithmetic` instead of `Base` in this branch.

Reply:
The function `f` is defined as `f(x) = x + 0.1`, so I guess this is actually the correct result?

Reply:
I think it is correct too (a bit confusing because of the global `setprecision` business though).

This is not the purpose of this PR so I think it is fine to let it like that for now, but I am quite confused by what is supposed to be tested here. If it just the fact that user define functions do not error and do some interval computation, then we could as well test
```julia
@test isa(f(d), Interval)
```